docs: add markAllAsTouched and clear method to doc (#30655)
angular/angular: aio/content/guide/reactive-forms.md

</div>

### Step 3: Marking all Form field as touched

> It is possible to mark all fields of Form using `markAllAsTouched()`. This is really useful in case you have multiple forms on your view and want to display validation message for both on a single button click.

Reviewer: Done.

</div>

### Step 5: Removing Controls from FormArray

> Once controls are added dynamically it also possible to remove the controls dynamically from the array using `removeAt()` function, which takes the index position and removes the control.

Reviewer 2: ```suggestion
Once controls are added to a `FormArray` dynamically, it is possible to remove them from the array using the `removeAt()` function. The function removes the control at the specified index.
```

Reviewer: Done.

<code-example path="reactive-forms/src/app/profile-editor/profile-editor.component.ts" region="remove-alias" header="src/app/profile-editor/profile-editor.component.ts (remove alias)">

</code-example>

### Step 6: Clear All Controls from FormArray

> It is possible to clear all controls from FormArray. You can use `clear()` function.

Reviewer 2: ```suggestion
It is possible to clear all controls from a FormArray. Call the `clear()` function to accomplish this.
```

Reviewer: Done.
